static/app/views/issueDetails/traceTimeline/traceIssue.tsx

```diff
@@ -81,14 +83,20 @@ export function TraceIssueEvent({event}: TraceIssueEventProps) {
 function getTitleSubtitleMessage(event: TimelineEvent) {
   let title;
   // XXX: This is not fully correct but it will make following PRs easier to review
-  const subtitle = event.transaction;
+  let subtitle = event.transaction;
   let message = event.message;
   try {
     title = event.title.trimEnd();
     if (event['event.type'] === 'error') {
       if (title[title.length - 1] !== ':') {
         title = event.title.split(':')[0];
       }
+    } else if (event['event.type'] === 'default') {
+      // https://github.com/getsentry/sentry/blob/f08644a004f9980d48f93dec2d7cfc9eeecd9a9e/static/app/utils/events.tsx#L179-L184
+      subtitle = '';
+      // https://github.com/getsentry/sentry/blob/f08644a004f9980d48f93dec2d7cfc9eeecd9a9e/static/app/utils/events.tsx#L59-L60
+      const errorEvent = event as TimelineDiscoverEvent;
```

Comment:
`culprit` is not shared by both types, thus, I must distinguish them here.

Reply:
It's better to fix the types in this case so that typescript can discriminate them without you needing to use `as`.

If TimelineDiscoverEvent has `event.type': 'default'` in the type def, typescript should do this automatically.

Reply:
I spoke with Malachi and I will be addressing this in a follow-up.